clusterloader2/testing/experimental/storage/pod-startup/ephemeral-volumes/cluster-wide/config.yaml

```diff
@@ -0,0 +1,101 @@
+# ASSUMPTIONS:
+# - Underlying cluster should have 100+ nodes.
+# - Number of nodes should be divisible by NODES_PER_NAMESPACE (default 100).
+
+#Constants
+# TODO(mucahitkurt) copied from load tests, open for discussion.
+{{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
+{{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
+{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 30}}
+
+{{$POD_TEMPLATE_PATH := DefaultParam .POD_TEMPLATE_PATH "../emptydir/pod_with_emptydir.yaml"}}
+{{$LOAD_TEST_THROUGHPUT := DefaultParam .LOAD_TEST_THROUGHPUT 10}}
+
+{{$VOLUMES_PER_POD := DefaultParam .VOLUMES_PER_POD 1}}
+{{$VOLUME_TEMPLATE_PATH := .VOLUME_TEMPLATE_PATH}}
+{{$PROVISION_VOLUME := DefaultParam .PROVISION_VOLUME false}}
+
+{{$APP_NAME := "pod-load"}}
+{{$GROUP := "pod-with-ephemeral-volume-startup-latency"}}
+
+#Variables
+# used stateless pod SLO for initial run
+{{$namespaces := DivideInt .Nodes $NODES_PER_NAMESPACE}}
+{{$podStartupTimeout := 5}}
+{{$totalPods := MultiplyInt $namespaces $NODES_PER_NAMESPACE $PODS_PER_NODE}}
+{{$podsPerNamespace := MultiplyInt $NODES_PER_NAMESPACE $PODS_PER_NODE}}
+{{$volumesPerNamespace := MultiplyInt $podsPerNamespace $VOLUMES_PER_POD}}
+
+name: pod-with-ephemeral-volume-startup-latency
+automanagedNamespaces: {{$namespaces}}
+tuningSets:
+- name: UniformQPS
+  qpsLoad:
+    qps: {{$LOAD_TEST_THROUGHPUT}}
+steps:
+# Start measurements
+- measurements:
+  - Identifier: APIResponsiveness
+    Method: APIResponsiveness
+    Params:
+      action: reset
+  - Identifier: PodWithMultiVolumeStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: start
+      labelSelector: group = {{$GROUP}}
+      threshold: {{$podStartupTimeout}}s
+{{ if $PROVISION_VOLUME }}
+# Provision volumes
+- phases:
+  - namespaceRange:
+      min: 1
+      max: {{$namespaces}}
+    replicasPerNamespace: {{$volumesPerNamespace}}
+    tuningSet: UniformQPS
+    objectBundle:
+    - basename: vol-{{$APP_NAME}}
+      objectTemplatePath: {{$VOLUME_TEMPLATE_PATH}}
+{{ end }}
+# Create pods
+- phases:
+  - namespaceRange:
+      min: 1
+      max: {{$namespaces}}
+    replicasPerNamespace: {{$podsPerNamespace}}
+    tuningSet: UniformQPS
+    objectBundle:
+    - basename: {{$APP_NAME}}
+      objectTemplatePath: {{$POD_TEMPLATE_PATH}}
```

Comment:
I'm fine with it as a starting phase, but we should move them to use deployments (using individual pods is more like anti-pattern, and startup-time is measured from pod creation anyway, so it shouldn't cause any additional problems).

Reply:
When I use a controller like deployments I can't control the volume names of the pod that controller creates. 

I tried it with statefulset,  for replication number 2, statefulset creates pod likes app-0-pod-0, app-0-pod-1, if I want to use volumes with these pod I should create volumes likes pod-0-vol-0, pod-0-vol-1, pod-1-vol-0, pod-1-vol-1. But I can't create volume names like 2 indexed values for pod-[podIndex]-vol-[volumeIndex], because clusterloader2 framework doesn't allow this.

I know that it's an anti-pattern working directly with the pods without a controller, but I couldn't solve the above problem and used directly pod.

Reply:
I opened https://github.com/kubernetes/perf-tests/issues/666 for it - let's continue the discussion there. I think I still don't understand where the problem is, but I don't want to block this PR on it.